osf/metadata/utils.py

```diff
@@ -23,6 +23,41 @@
 }
 
 
+def datacite_format_creators(creators):
+    """ Format a list of contributors to match the datacite schema
+
+    :param contributors_list: list of OSFUsers to format
+    :return: formatted json for datacite
+    """
+    creators_json = []
+    for creator in creators:
+        name_identifiers = [
+            {
+                'nameIdentifier': f'{creator._id}/',
+                'nameIdentifierScheme': 'OSF',
+                'schemeURI': settings.DOMAIN
+            }
+        ]
+
+        if creator.external_identity.get('ORCID'):
+            verified = list(creator.external_identity['ORCID'].values())[0] == 'VERIFIED'
+            if verified:
+                name_identifiers.append({
+                    'nameIdentifier': list(creator.external_identity['ORCID'].keys())[0],
+                    'nameIdentifierScheme': 'ORCID',
+                    'schemeURI': 'http://orcid.org/'
+                })
+
+        creators_json.append({
```

Comment:
Do creators have a contributorType, too? Can you add some module-level documentation that links to the spec you're using to format these?